examples/tutorials/DeepChemXAlphafold.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>Why do we need this job name? Give a bit more context</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>